src/openssl_lib.cpp

```diff
@@ -910,6 +910,31 @@ int OpenSSLLib::SSL_EVP_CIPHER_CTX_set_padding(EVP_CIPHER_CTX * c, int pad) noex
 {
     return EVP_CIPHER_CTX_set_padding(c, pad);
 }
+int OpenSSLLib::SSL_PKCS5_PBKDF2_HMAC(const char *pass, int passlen,
+                                      const unsigned char *salt, int saltlen, int iter,
+                                      const EVP_MD *digest, int keylen, unsigned char *out) noexcept
+{
+    return PKCS5_PBKDF2_HMAC(pass, passlen, salt, saltlen, iter, digest, keylen, out);
+}
+
+
+int OpenSSLLib::SSL_ECDH_KDF_X9_63(unsigned char *out, size_t outlen, const unsigned char *Z, size_t Zlen,
+                                   const unsigned char *sinfo, size_t sinfolen, const EVP_MD *md) noexcept
+{
+    /*
+     * The old name for ecdh_KDF_X9_63 (include/openssl/ec.h:1110)
```

Comment:
still not sure how to handle this, maybe we should really use https://www.openssl.org/docs/manmaster/man3/EVP_KDF_derive.html instead using a (sonnish to be) deprecated function. On the other hand this is what ecdh_KDF_X9_63/2 uses anyway...

Reply:
As discussed we will resolve this issue.